_docs_integrate/version-5.md

```diff
@@ -99,6 +99,69 @@ In addition, in contrast to the RequestItems, a RequestItemGroup did not have to
 The `mustBeAccepted` property of the RequestItems will be kept.
 {: .notice--info}
 
+## New mandatory wrappers for non-standard content of RelationshipTemplates, Relationships and Messages
```

Comment:
RelationshipCreationContent missing

Reply:
I wanted to have the sections as isolated as possible, so no creation content in the heading

Reply:
So you want to talk about relationships in the title instead of the creationContent inside? I'm ok with that.

Reply:
Do you see anything left to do here?

Reply:
FYI, the PR has now been taken over by me @jkoenig134 :D This comment can be resolved, right?

Reply:
yep